### PR TITLE
Added additional info about storage capacity and backup restores

### PR DESCRIPTION
**Pull request by mijanss**
Fixes elastx/issue-tracker#202

### CODE REVIEW

**Comment by testviking** [comment]
Should we add something like:

Please remember that the database becomes read-only when 90% or more of the disk space is used.

**Comment by testviking** [comment]
closes elastx/issue-tracker/issues/202

**Review by Tipset — CHANGES_REQUESTED** [review]
Do the proposed changes from @testviking before pushing.

**Review by Tipset — APPROVED** [review]
This looks better.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md [hunk 1]
```diff
@@ -36,6 +37,9 @@ For example, if you create a datastore with PostgreSQL 11, then all the database
 #### Backup copy temporarily stored on the node
```
Comment (testviking) [inline content/en/docs/dbaas/getting-started/DBaaS-guides.md:37]: This limitation has been fixed by S9S and can be removed from here.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ For example, if you create a datastore with PostgreSQL 11, then all the database
...
 A datastore backup copy gets temporarily stored locally on the node before being uploaded to object storage. This will have an effect on capacity needs for a datastore.
 
+#### Restore from backup and available storage capacity
+When you wish to restore a datastore from backup, you need to ensure that your storage usage on each datastore node is at maximum 65%. This is because the current restore from backup method will use your datastore nodes storage to temporarily store the backup files during the restore. To ensure this, you need to use the Scale Storage functionality when needed, before doing a restore from backup. After restore from backup is done, you can resize the datastore nodes back to the desired size again.
```
[the remark this answers is not in the document]
Comment (testviking) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md:41]: The text seems to be clear now. I cannot resolve this. Someone else will need to resolve it.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md [hunk 3]
```diff
@@ -36,6 +37,9 @@ For example, if you create a datastore with PostgreSQL 11, then all the database
 #### Backup copy temporarily stored on the node
 A datastore backup copy gets temporarily stored locally on the node before being uploaded to object storage. This will have an effect on capacity needs for a datastore.
 
+#### Restore from backup and available storage capacity
```
Comment (testviking) [inline content/en/docs/dbaas/getting-started/DBaaS-guides.md:40]: Maybe the heading should be something more like:

Before you restore from backup: storage usage must be max 65%

**Comment by testviking** [comment]
I added two more reviews.

**Comment by testviking** [comment]
With my permission level in this repo, all my reviews get stuck as Pending and I cannot resolve them myself. So please keep this in mind.

**Review by testviking — COMMENTED** [review]
Please see my comments.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (testviking) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md:41]: I am fine with leaving it at 65%. I don't know if we need to ever think about these details.

**Comment by testviking** [comment]
My bad. I started a review now. It has been a while, so I forgot that I need to start a review.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md (hunk 1 above)
Comment (mijanss) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md]: Fixed limitation removed.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md (hunk 3 above)
Comment (mijanss) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md]: Changed the heading.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mijanss) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md:38]: Thank you @testviking  for the detailed clarification.
I agree that 65% could be kept.

**Review by testviking — APPROVED** [review]
Thank you. Looks good.

File: content/en/docs/dbaas/getting-started/DBaaS-guides.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tipset) [reply content/en/docs/dbaas/getting-started/DBaaS-guides.md:38]: nice @testviking , I suppose this is now resolved.

**Pull request merged**